SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -2187,6 +2187,7 @@
                 "helm",
                 "hvloader",
                 "hvloader-signed",
+                "ig",
```

Comment:
Why do we want to add this package? I've heard the name "Inspektor Gadget" floating around but I can't find the right deliverable or task right now.

Reply:
We do have a workitem for it: [link](https://dev.azure.com/microsoft/OS/_workitems/edit/49010795)